diff --git a/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/types/CellularGeneration.java b/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/types/CellularGeneration.java
index 8b56484..71737dc 100644
--- a/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/types/CellularGeneration.java
+++ b/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/types/CellularGeneration.java
@@ -26,6 +26,7 @@ public enum CellularGeneration {
     }
 
     @Nullable
+    @SuppressWarnings("deprecation")
     public static CellularGeneration fromNetworkInfo(@Nullable NetworkInfo networkInfo) {
         if (networkInfo == null) {
             return null;
