diff --git a/node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java b/node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java
index e1c71e2..1fa06a0 100644
--- a/node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java
+++ b/node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java
@@ -48,6 +48,7 @@ import javax.annotation.Nullable;
  * {@link ReactDialogPickerManager} components. These are merged back on the JS side into one
  * React component.
  */
+@SuppressWarnings("deprecation")
 public abstract class ReactPickerManager extends BaseViewManager<ReactPicker, ReactPickerShadowNode> {
   private static final ReadableArray EMPTY_ARRAY = Arguments.createArray();
 
